docs/model_library/operational_water_management/index.rst

```diff
@@ -430,108 +430,108 @@ For each completions pad and time period, the volume of water (excluding freshwa
 
```

Comment:
For more extra credit, you can change line 427 to

    **Completions Pad Processing Capacity:** :math:`\forall \textcolor{blue}{p \in CP}, \textcolor{blue}{t \in T}`